spark-jobs/src/main/scala/filodb/repair/ChunkCopier.scala

```diff
@@ -62,24 +62,30 @@ class ChunkCopier(conf: SparkConf) {
   val sourceCassConfig = sourceConfig.getConfig("cassandra")
   val targetCassConfig = targetConfig.getConfig("cassandra")
 
+  val isDownsampleCopy = conf.getBoolean("spark.filodb.chunks.copier.is.downsample.copy", false)
+
   val datasetName = conf.get("spark.filodb.chunks.copier.dataset")
-  val datasetRef = DatasetRef.fromDotString(datasetName)
   val targetDatasetConfig = datasetConfig(targetConfig, datasetName)
+  val datasetRef = if (isDownsampleCopy) {
+    val downsampleResolution = Duration(conf.get("spark.filodb.chunks.copier.dataset.downsample.resolution"))
+    DatasetRef(s"${datasetName}_ds_${downsampleResolution.toMinutes}")
+  } else {
+    DatasetRef.fromDotString(datasetName)
+  }
 
   // Examples: 2019-10-20T12:34:56Z  or  2019-10-20T12:34:56-08:00
   private def parseDateTime(str: String) = Instant.from(DateTimeFormatter.ISO_OFFSET_DATE_TIME.parse(str))
 
-  val ingestionTimeStart = parseDateTime(conf.get("spark.filodb.chunks.copier.repairStartTime"))
-  val ingestionTimeEnd = parseDateTime(conf.get("spark.filodb.chunks.copier.repairEndTime"))
+  val copyStartTime = parseDateTime(conf.get("spark.filodb.chunks.copier.start.time"))
+  val copyEndTime = parseDateTime(conf.get("spark.filodb.chunks.copier.end.time"))
```

Comment:
It is good to keep the "ingestion" term around since the time ranges are ingestion time related. Perhaps use `ingestionTimeStartForCopy` and `ingestionTimeEndForCopy` ?